packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -71,57 +69,65 @@ export const PaymentConfirmation = ({
   const downloadUrl: URL = new URL(productUrl);
   downloadUrl.searchParams.append('email', email);
 
+  // TW classes
+  const subheadingClasses = 'text-grey-400 max-w-sm';
+  const h2classes = 'text-xl font-normal mx-0 mt-6 mb-3';
+  const h3classes = 'text-sm font-semibold';
+  const bottomRowClasses = 'flex justify-between items-center text-grey-400';
+
   return (
     <>
       <SubscriptionTitle screenType="success" />
       <section
-        className={`container card payment-confirmation ${className}`}
+        className="container payment-confirmation"
```

Comment:
I noticed the width of this is different than the original, should we be removing `container` and keeping `card instead?

Reply:
`container` removed and styles updated to match `SubscriptionTitle` spacing! 👍 

Regarding `card`, I couldn't find any styles associated with it in the inspector, and after doing a search, only found one reference - `#main-content.card` - that doesn't affect this component, so I removed it.

Note: I did update the width of the paragraph below `h2` to the tailwind utility class `.max-w-sm` (i.e. 24rem, or 384px) as opposed to what was originally set in the stylesheet `max-width: 400px;`. The alternative is `.max-w-md` (i.e. 28rem, or 448px), but opted for the smaller width since it's closer to the original and also better on the eyes and for a11y. 🙂

Reply:
Thanks for digging into this! Yep I noticed that you updated that - I am aligned, it looks better!